api/src/test/scala/org/globalnames/resolver/api/QueryParserSpec.scala

```diff
@@ -1,6 +1,9 @@
-package org.globalnames.resolver.api
+package org.globalnames
+package resolver
+package api
 
 import org.scalatest.FunSpec
+import org.scalatest.Matchers._
 import org.globalnames.resolver.api.QueryParser.{Modifier, SearchPart}
```

Comment:
We really don't need `import` that long. It can be shorten to `import QueryParser.{Modifier, SearchPart`.